source/tutorials/pd-mice.rst

```diff
@@ -0,0 +1,748 @@
+|                         | the human who      |                 |                  |
+|                         | donated the        |                 |                  |
+|                         | feces              |                 |                  |
++-------------------------+--------------------+-----------------+------------------+
+| donor_status            | whether the        | categorical     | ``"Healthy"``;   |
+|                         | donor has          |                 | ``"PD"``         |
+|                         | Parkinson's        |                 |                  |
+|                         | disease or not     |                 |                  |
+|                         | (Donor             |                 |                  |
+|                         | ``pd_1`` had       |                 |                  |
+|                         | Parkinson's        |                 |                  |
+|                         | disease;           |                 |                  |
+|                         | ``hc_1``           |                 |                  |
+|                         | was                |                 |                  |
+|                         | neurologically     |                 |                  |
+|                         | healthy)           |                 |                  |
++-------------------------+--------------------+-----------------+------------------+
+| days_post_transplant    | the number of      | numeric         | 7, 14, 21, 49    |
+|                         | days after the     |                 |                  |
+|                         | mice were          |                 |                  |
+|                         | humanized          |                 |                  |
++-------------------------+--------------------+-----------------+------------------+
+
+
+Even though the mouse ID looks like a number, we will specify the type using the ``#q2_type`` column in the data.
+
+The metadata is avaliable as a `Google Sheet`_, or ou can download it directly from and save it as a tsv.
+
+.. download::
+   :url: https://data.qiime2.org/2019.7/tutorials/pd-mice/sample_metadata.tsv
+   :saveas: metadata.tsv
+
+The sample metadata will be used through out the tutorial.
+
+Loading the data into QIIME 2
+=============================
+
+In QIIME 2, all data is structured as an Artifact of a specific semantic type. The artifacts contain the data as well as information about the data, including a record of the original data, the tools used to process it. This allows for better tracking of how you actually got to where you are in your analysis. You can learn more about common QIIME 2 Artifacts and types of artifacts :doc:`here <../semantic-types/>`.
+
+Our samples were amplified u|sing the `EMP 515f-806r`_ primers and sequenced on an Illumina MiSeq with a 2x150bp kit. The hypervariable
```

Comment:
```suggestion
Our samples were amplified using the `EMP 515f-806r`_ primers and sequenced on an Illumina MiSeq with a 2x150bp kit. The hypervariable
```

Reply:
I am so proud of myself that this is the first major spelling mistake at line 122